tests/middleware/test_logging.py

```diff
@@ -175,3 +231,25 @@ async def app(scope, receive, send):
             if record.name == "uvicorn.access"
         ]
         assert '"GET / HTTP/1.1" 599' in messages.pop()
+
+
+@pytest.mark.anyio
+@pytest.mark.parametrize("use_colors", [True, False])
```

Comment:
The goal of improving coverage is not to blindly improve it...

It doesn't look like we have a big motivation for creating this test, based on the name it has.

Maybe we can create a test for: "test_port_zero()` or something like that...

Please create another PR for this. :eyes:

Reply:
There were no tests for servers that have port 0. That's the reason I had added it. I initially added it in test_main, later decided to check the logs to ensure a port is assigned. 

I can move this to another PR and probably rename it.

Reply:
Done, I have removed other tests. Will create another PR for those.